### DISCUSSION TITLE
Unable to add opacity to tor browser

### DISCUSSION THREAD

**Discussion by jenil1122** [prasanthrangan/hyprdots#1160] [Q&A]
i tried the following code : 
```
windowrulev2 = opacity 0.80 0.80,class:^(Tor-Browser)$
```
with differnt variations such as 
1)Tor-Browser
2)Tor
3)tor-browser
**Nothing Worked**

**Comment by mahaveergurjar** [comment]
Type this command in terminal 
```shell
hyprctl client 
``` 
See the tor browser class 
And then add the proper tor browser class 
In this line 
```shell
windowrulev2 = opacity 0.80 0.80,class:^(Tor-Browser)$ 
``` 
Replace Tor-Browser with tor browser class
